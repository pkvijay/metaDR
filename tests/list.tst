list /
create /test 123
create /test/a 456
create /test/b 789
list /test
remove /test/b
list /test
list /missing_key
list /
