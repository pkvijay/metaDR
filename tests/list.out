CREATED
CREATED
CREATED
a
b
REMOVED
a
KEY NOT FOUND
test
